eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```diff
@@ -54,6 +54,21 @@ jobs:
       shouldContinueOnError: true
       alwaysRun: true
 
+  #ToDo: add AOT version when this is confirmed to work
+  #ToDo: confirm if adding it here is fine
+  - template: /eng/pipelines/common/templates/wasm-library-tests.yml
```

Comment:
Is this expected to be stable right now?

Reply:
If not, then we should add only to `runtime-wasm*` for now.

Reply:
> The `runtime-wasm-libtests` are run as part of rolling build?

`runtime-wasm-libtests` is a separate manual-run-only pipeline. Some of the jobs are part of `runtime-extra-platforms` though, and those run for rolling builds.

Reply:
"Is this expected to be stable right now?" I only know that for iOS it's not. For browser it might be, we'll see in this PR.